tesseract-server/src/handlers/aggregate.rs

```diff
@@ -120,11 +123,12 @@ pub fn do_aggregate(
         .responder()
 }
 
-#[derive(Debug, Deserialize, Serialize)]
+#[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct AggregateQueryOpt {
-    drilldowns: Option<Vec<String>>,
-    cuts: Option<Vec<String>>,
-    measures: Option<Vec<String>>,
+    pub drilldowns: Option<Vec<String>>,
+    pub cuts: Option<Vec<String>>,
+    pub measures: Option<Vec<String>>,
+    pub year: Option<String>,
```

Comment:
If we're integrating the year latest stuff with logic layer, this will get moved to another endpoint.